cmd/process-agent/collector.go

```diff
@@ -285,6 +300,10 @@ func (l *Collector) run(exit chan struct{}) error {
 	for _, e := range l.cfg.Orchestrator.OrchestratorEndpoints {
 		orchestratorEps = append(orchestratorEps, e.Endpoint.String())
 	}
+	eventsEps := make([]string, 0, len(processEventsAPIEndpoints))
+	for _, e := range processEventsAPIEndpoints {
+		eventsEps = append(eventsEps, e.Endpoint.String())
+	}
```

Comment:
Nit: you may want to assign here instead of appending

```
    eventsEps := make([]string, len(processEventsAPIEndpoints))
	for i, e := range processEventsAPIEndpoints {
		eventsEps[i] = e.Endpoint.String()
	}
```

Reply:
Since we're using this pattern for `eps` and `orchestratorEps` above `eventsEps`, I'll keep it as it is for now to keep consistency.
https://github.com/DataDog/datadog-agent/blob/65854dd66094dd278aa5ae0bbd6f44b6576ca079/cmd/process-agent/collector.go#L295-L306